buildconfig/pygame-stubs/math.pyi

```diff
@@ -272,6 +274,8 @@ class Vector3:
     def as_spherical(self) -> Tuple[float, float, float]: ...
     def from_spherical(self, spherical: Tuple[float, float, float]) -> None: ...
     def copy(self) -> Vector3: ...
+    def move_towards(self) -> Vector3: ...
+    def move_towards_ip(self) -> None: ...
```

Comment:
the function signatures need to indicate the first 'target' argument and the 'distance' argument